pkgs/tools/wayland/wlrctl/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromSourcehut, cmake, meson, pkg-config, libxkbcommon, wayland, scdoc, ninja }:
+
+stdenv.mkDerivation rec {
+  pname = "wlrctl";
+  version = "0.2.1";
+
+  src = fetchFromSourcehut {
+    owner = "~brocellous";
+    repo = "wlrctl";
+    rev = "v${version}";
+    sha256 = "039cxc82k7x473n6d65jray90rj35qmfdmr390zy0c7ic7vn4b78";
+  };
+  dontUseCmakeConfigure = true;
+  nativeBuildInputs = [ cmake meson pkg-config scdoc ninja ];
```

Comment:
Do we really need ninja, meson, cmake and pkg-config?

Reply:
removed cmake